precompiles/utils/src/solidity/codec/xcm.rs

```diff
@@ -38,7 +38,7 @@ pub const JUNCTION_SIZE_LIMIT: u32 = 2u32.pow(16);
 // The first byte represents the enum variant to be used.
 // 		- Indexes 0,2,3 represent XCM V2 variants
 // 		- Index 1 changes name in V3 (`ByGenesis`), but is compatible with V2 `Named`
-// 		- Indexes 4~10 represent new XCM V3 variants
+// 		- Indexes 4~11 represent new XCM V3 variants
```

Comment:
Was this new variant added in XCM v4? If so, we should add a new comment indicating that index 11 comes from XCM v4.

Reply:
It was added to the V3 variant: https://github.com/paritytech/polkadot-sdk/pull/2517/files